web/apps/labelstudio/src/pages/Home/HomePage.tsx

```diff
@@ -0,0 +1,195 @@
+import type { Page } from "../types/Page";
+import { Button } from "@humansignal/shad/components/ui/button";
+import { IconFolder, SimpleCard, Spinner } from "@humansignal/ui";
+import { IconExternal, IconFolderAdd, IconUserAdd } from "@humansignal/icons";
+import { HeidiTips } from "../../components/HeidiTips/HeidiTips";
+import { useQuery } from "@tanstack/react-query";
+import { useAPI } from "../../providers/ApiProvider";
+import { useState } from "react";
+import { CreateProject } from "../CreateProject/CreateProject";
+import { InviteLink } from "../Organization/PeoplePage/InviteLink";
+import { Heading, Sub } from "@humansignal/typography";
+
+const resources = [
+  {
+    title: "Documentation",
+    url: "https://labelstud.io/guide/",
+  },
+  {
+    title: "API Documentation",
+    url: "https://api.labelstud.io/api-reference/introduction/getting-started",
+  },
+  {
+    title: "LabelStud.io Blog",
+    url: "https://labelstud.io/blog/",
+  },
+  {
+    title: "Slack Community",
+    url: "https://slack.labelstud.io",
+  },
+];
+
+const actions = [
+  {
+    title: "Create Project",
+    icon: IconFolderAdd,
+    type: "createProject",
+  },
+  {
+    title: "Invite People",
+    icon: IconUserAdd,
+    type: "invitePeople",
+  },
+] as const;
+
+type Action = (typeof actions)[number]["type"];
+
+export const HomePage: Page = () => {
+  const api = useAPI();
+  const [creationDialogOpen, setCreationDialogOpen] = useState(false);
+  const [invitationOpen, setInvitationOpen] = useState(false);
+  const { data, isFetching, isSuccess, isError } = useQuery({
```

Comment:
Noting that `useQuery` calls should be lifted out of components, no action required in this PR, just a talking point for FE Guild 🙂